### PR TITLE
Support overriding the `nixpkgs` to use.

### PR DESCRIPTION
**Pull request by nc6**
This argument was present before, but was not respected.

### CODE REVIEW

**Comment by nc6** [comment]
Again, based atop previous PR, but could be isolated if desired.

**Comment by 4e6** [comment]
Do you mind if we drop previous PR in favor of this one? Since they are incremental. I'm just too lazy to keep up with both :)

File: src/Distribution/Nixpkgs/Haskell/Stack/PrettyPrinting.hs [hunk 1]
```diff
@@ -74,10 +75,11 @@ overrideHaskellPackages oc packages = vcat
     , "};"
     , ""
     ]
-  , "in callPackage <nixpkgs/pkgs/development/haskell-modules> {"
+  , "in callPackage (nixpkgsPath + \"/pkgs/development/haskell-modules\") {"
```
Comment (nc6) [inline src/Distribution/Nixpkgs/Haskell/Stack/PrettyPrinting.hs]: Aha, useful. I'll move to that.

**Comment by nc6** [comment]
Sounds sensible!

**Comment by nc6** [comment]
@4e6 Addressed comments.

**Comment by 4e6** [comment]
Thanks! Seems all good now.

**Pull request merged**